Automattic/jetpack
# Bump height of the connect-in-place iframe to fit error messages

Bump height of the connect-in-place iframe to fit error messages.

Fixes no known issue. See D32584-code for screenshots.

#### Changes proposed in this Pull Request:
* Bump height of the connect-in-place iframe to fit error messages.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* p1HpG7-7nj-p2

#### Testing instructions:
- Run this branch
- Confirm that the iframe height is big enough to fit the content in each stage of the connection and in case of an error.

#### Proposed changelog entry for your changes:
None.
